lsp/src/main/java/com/salesforce/slds/lsp/diagnostics/Diagnoser.java

```diff
@@ -101,4 +102,16 @@ public Bundle getBundle(TextDocumentItem entry) throws IOException {
         return bundle;
     }
 
+    private static boolean isLightningComponentFiles(File file) {
+        String fileName = file.getName();
+        int position = fileName.lastIndexOf('.');
+        if (position != -1) {
+            String fileExtension = fileName.substring(position);
+            return SUPPORTED_FILE_EXTENSIONS.contains(fileExtension.toLowerCase());
+        }
+
+        return false;
+    }
+
+    private static List<String> SUPPORTED_FILE_EXTENSIONS = ImmutableList.of(".html", ".js", ".css", ".cmp", ".app");
```

Comment:
In what case are folks using .app files for lwc apps?

Reply:
`.app` is for Aura Application

Reply:
Great thanks.